### DISCUSSION TITLE
Validate answer as correct/incorrect without showing the actual answer

### DISCUSSION THREAD

**Discussion by 4lex1v** [SethClydesdale/genki-study-resources#259] [Ideas]
こんにちは、
While working through exercises, I was thinking that instead of showing the answer, I would much prefer a mode that would just tell me if the provided answer is correct or not (when I manually type the answer), without showing the answer. This would help me with forcing myself to analyze the answer and understand why it's wrong, rather than just look at the answer. Perhaps an option in the settings to enable dynamic input checking that would highlight the input field with red if it's wrong or green if it's correct would be helpful, to make the experience smoother, i.e no need to scroll after each input to press the check answer button.

Thank you!

**Comment by SethClydesdale** [comment]
Hello!

That's a pretty nice idea and should be possible to implement by checking the answer validity upon input. I'll put it on my list of things to experiment with when I've got free time 👍
